tests/Immutable.lua

```diff
@@ -0,0 +1,141 @@
+--[[
```

Comment:
I think this should be replaced with Cryo

Reply:
Sure. The current Signal is just lifted from CorePackages

Reply:
Cyro would just have to be dropped in like Immutable since it would have to be lifted from CorePackages

Reply:
Maybe it would be better to wait until Cyro is available on github since it's more than one file?

Reply:
Signal is fine to stay. I'm more concerned with Cryo since I see it as the future of both Immutable and Functional in CorePackages. I'm fine with just using Immutable until we have a proper Cryo release though. Easier drop-in this way.